src/sentry/tasks/integrations/migrate_ignored_fields.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any
+
+from sentry.models import OrganizationIntegration
+from sentry.tasks.base import instrumented_task
+from sentry.tasks.integrations import logger
+
+
+@instrumented_task(
+    name="sentry.tasks.integrations.migrate_ignored_fields",
+    queue="integrations",
+    default_retry_delay=60 * 5,
+    max_retries=5,
+)
+def migrate_ignored_fields(
+    integration_id: int,
+    plugin_ignored_fields: list,
+    logging_context: Any,
+) -> None:
```

Comment:
What is this task doing? I'm don't think I have enough context on Jira and the plugins, but what's being migrated here? Also does this need to be a task, since it looks like it's pretty synchronous code, unless I'm missing something.

Reply:
Lol I left a whole comment about it: https://github.com/getsentry/sentry/pull/37577#discussion_r940704117 but yeah I guess it doesn't need to be async